### PR TITLE
Resolves #79 - Update to 7.4 SDK

### LINKED ISSUE #79: 7.4 issue
I decided to be your sacrificial guinea pig, and upgrade my switcher and control software to 7.4. 

Can confirm that the OSC Addresses menu is doing the same thing as 2.5 was doing with 7.3.

Assuming the build just needs to be updated to 7.4 support, lmk if I can do anything to help with this, diff etc. Havent gotten into testing if actual control still works yet.

### PR DESCRIPTION
**Pull request by SteffeyDev**
Hopefully should fix sources for 7.4.  Need to verify functionality still.

### CODE REVIEW

**Comment by carbon43** [comment]
Word, I have to run out but can test a bit later tonight. Also before I try it I'll see if I have control still under 7.4 + 2.5.2. I saw some folks didnt have control in the last scenario like this, but even though that address page was partially blank I still had control (7.3 + 2.5). So curious if thats still the case here. Once thats tested I'll roll to this and see what happens.

**Comment by SteffeyDev** [comment]
@sneat Ok, Readme is updated

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# AtemOSC v2.5.2
+# AtemOSC v2.5.3
 
 ## Features
 This is a macOS application, providing an interface to control an ATEM video switcher via OSC. 
 
 ![atemOSC](https://github.com/danielbuechele/atemOSC/raw/master/atemOSC.jpg)
 
-The current version is built for Mac OS 10.13 (as of version 2.5.2). A compiled and runnable version of the atemOSC is included which has been built against Blackmagic SDK 7.3 (as of version 2.5.2). 
+The current version is built for Mac OS 10.13 (as of version 2.5.3). A compiled and runnable version of the atemOSC is included which has been built against Blackmagic SDK 7.4 (as of version 2.5.3).
```
Comment (sneat) [inline README.md]: Sorry to be a pain, but the `built for Mac OS 10.13` section can stay until we build it for a new version. Only the `Blackmagic SDK` bit needs to change.

Comment (SteffeyDev) [reply README.md]: Ah, that makes more sense.

**Comment by SteffeyDev** [comment]
Ok, changed from 2.5.2 to 2.5 for build version because I now understand intentions and it should have been 2.5 to start with.

**Pull request merged**